subsys/caf/events/sensor_data_aggregator_event.c

```diff
@@ -12,7 +12,7 @@ static void log_sensor_data_aggregator_event(const struct app_event_header *aeh)
 {
 	const struct sensor_data_aggregator_event *event = cast_sensor_data_aggregator_event(aeh);
 
-	APP_EVENT_MANAGER_LOG(aeh, "Send sensor buffer desc: %s", event->sensor_descr);
+	APP_EVENT_MANAGER_LOG(aeh, "Send sensor buffer desc address: %p", event->sensor_descr);
```

Comment:
btw - maybe in the end the load thread should have higher priority compared to logger... just wondering